packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -132,6 +132,18 @@
 					<div v-if="option.description" class="option-description" v-html="getOptionsOptionDescription(option)"></div>
 				</div>
 			</n8n-option>
+			<n8n-option
+				v-if="isSupportedByHttpRequestNode && ['resource', 'operation'].includes(parameter.name)"
```

Comment:
can we simply inject this option into parameter json we are passing down instead?